ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -808,19 +819,17 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
           ErrorCategory.InvalidGivenCurrentSystemStateOther, // It may succeed at a later time
         ) {
       case class RejectEnriched(
-          message: String,
+          override val cause: String,
           ledger_time: Instant,
           ledger_time_lower_bound: Instant,
           ledger_time_upper_bound: Instant,
       )(implicit loggingContext: ContextualizedErrorLogger)
-          extends LoggingTransactionErrorImpl(
-            cause = s"Invalid ledger time: $message"
-          )
+          extends LoggingTransactionErrorImpl(cause = cause)
```

Comment:
Why do you need this assignment? Doesn't `LoggingTransactionErrorImpl(cause)` suffice?

Reply:
usually I add named arguments when the args list has default values for some arguments.